### PR TITLE
docs: add YAML as examples

### PR DESCRIPTION
**Pull request by FoSix**
## Description

Add YAML formatted examples next to Python code

## Motivation and Context

see #97 

## How Has This Been Tested?

Tested on a local pan.dev build

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: file
Comment (alperenkose) [inline file:1]: I think this file is added accidentally ?

**Review by alperenkose — COMMENTED** [review]
It looks good. I'm thinking if we shall also include a reference to the pan-os ansible collection, otherwise it might not make sense for the users why yaml examples are provided.

**Comment by FoSix** [comment]
> It looks good. I'm thinking if we shall also include a reference to the pan-os ansible collection, otherwise it might not make sense for the users why yaml examples are provided.

Yes, but the question is where. We have [this](https://pan.dev/panos/docs/#:~:text=easy%20to%20wrap%20it%20in%20an%20Ansible) text. Maybe somewhere here?

On the other hand, we already have [this](https://pan.dev/panos/docs/panos-upgrade-assurance/#:~:text=The%20libraries%20were%20written%20to%20support%20Ansible%20and%20XSOAR%20integrations.%20They%20depend%20on%20pan%2Dos%2Dpython%20libraries%20and%20therefore%20are%20quite%20easy%20to%20fit%20into%20the%20PanOS%20Ansible%20modules%20collection.)

File: file
Comment (FoSix) [reply file:1]: fixed

**Comment by FoSix** [comment]
> > Yes, but the question is where. We have [this](https://pan.dev/panos/docs/#:~:text=easy%20to%20wrap%20it%20in%20an%20Ansible) text. Maybe somewhere here?
> > On the other hand, we already have [this](https://pan.dev/panos/docs/panos-upgrade-assurance/#:~:text=The%20libraries%20were%20written%20to%20support%20Ansible%20and%20XSOAR%20integrations.%20They%20depend%20on%20pan%2Dos%2Dpython%20libraries%20and%20therefore%20are%20quite%20easy%20to%20fit%20into%20the%20PanOS%20Ansible%20modules%20collection.)
> 
> Exactly, I think we can add it following that line on the main page, sth like below perhaps?
> 
> > They depend on [pan-os-python](https://pan.dev/panos/docs/panospython/) libraries and therefore are quite easy to fit into the [PanOS Ansible modules collection](https://galaxy.ansible.com/paloaltonetworks/panos). _Equivalent yaml configurations are provided in the documentation as a reference to use with the PanOS Ansible modules collection._

I've added that line.

**Review by alperenkose — APPROVED** [review]

**Pull request merged**